ui/app/src/utils/i18n-legacy.ts

```diff
@@ -1031,6 +1063,20 @@ const localeSelectorControlMessages = defineMessages({
   }
 });
 
+const clusterAdminMessages = defineMessages({
+  clusterDeleted: {
+    id: 'clusterAdminMessages.clusterDeleted',
+    defaultMessage: '{cluster} deleted.'
+  }
+});
+
+const sitesViewMessages = defineMessages({
+  siteEdited: {
+    id: 'globalMenu.siteEdited',
+    defaultMessage: 'Site Edited.'
+  }
+});
```

Comment:
The siteEdited message still has an id that's not appropriate. 

Seeing that we have more than just the site edited (i.e. cluster deleted), add them both to a single object called `adminDashboardMessages`. Their **individual ids** should be for their own module, though (e.g. sitesAdmin.siteUpdated, clusterAdmin.clustedDeleted).